plugins/overlays/download.js

```diff
@@ -79,12 +93,6 @@ class DownloadOverlay extends React.Component {
     );
   }
 
-  _onAccept(){
-    const { handleSuccess, handleError, handleComplete } = this.props;
```

Comment:
if these are not used, they need to be removed from being passed into the overlay component